examples/19_local_datasource.py

```diff
@@ -29,8 +29,7 @@
 """
 # /// script
 # dependencies = [
-#   "earth2studio @ git+https://github.com/NVIDIA/earth2studio.git@0.11.0",
-#   "aurora",
```

Comment:
Wait it doesn't need aurora group now?

Reply:
I messed up, reason it was passing was because i already had Microsoft aurora in my python env